otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -277,9 +277,18 @@ def _train_model(
 
         validate = bool(cfg.data.get("val", None))
 
-        if self._hyperparams.learning_parameters.auto_decrease_batch_size:
+        if (
```

Comment:
Does the same code need to be scattered across each task? 
Couldn't it just be declared in a commonly used task?

Reply:
It's hard to do, because a few things are different depending on tasks. For example, action task should set True to `distributed` argument. And each task has different training function (e.g. train_detector, train_segmentor, etc.).